src/main/java/db/StorageTotalBalance.java

```diff
@@ -0,0 +1,8 @@
+package db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class StorageTotalBalance {
```

Comment:
**Total** and **Balance** words do not add anything useful to this class name.
Storage or FruitStorage is better.
```suggestion
public class Storage {
```